### PR TITLE
Expose qpext aggregate metrics port on container

### LINKED ISSUE #3241: expose prometheus metric port in qpext container
/kind feature

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]
on isvc pod, qpext container does not have metric port. i want to scrape kserve container metric using podMonitor.

when I try to scrape isvc metric using podMonitor, prometheus cannot scrape it.
because podMonitor's [PodMetricsEndpoint](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.PodMetricsEndpoint) should be port's name, not port integer. since qpext does not expose port 9088, there is no port name to put in podMonitor

CleanShot 2023-11-10 at 22 19 34@2x

### PR DESCRIPTION
**Pull request by sivanantha321**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3241

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:

```release-note
Expose qpext aggregate metrics port on container
```

### CODE REVIEW

File: qpext/README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ spec:
   predictor:
     sklearn:
       protocolVersion: v2
-      storageUri: "gs://seldon-models/sklearn/iris"
+      storageUri: "gs://seldon-models/sklearn/mms/lr_model"
```
Comment (yuzisun) [inline qpext/README.md]: change to kserve storage uri ?

Comment (sivanantha321) [reply qpext/README.md]: done

**Review by rachitchauhan43 — APPROVED** [review]
LGTM

**Pull request merged**